src/incubator/TextField/index.tsx

```diff
@@ -108,7 +107,7 @@ const TextField = (props: InternalTextFieldProps) => {
             validate={others.validate}
             validationMessage={others.validationMessage}
             validationMessageStyle={validationMessageStyle}
-            retainSpace={retainTopMessageSpace}
```

Comment:
what's the purpose of this change?

Reply:
To avoid calling `isEmpty` and `trim` on every render  if validation message is at the bottom (which is the default). Do you think this is too small of a change worth having?